app/main/controllers/template/rtmedia-functions.php

```diff
@@ -1042,6 +1042,14 @@ function rtmedia_edit_allowed() {
 		$flag = is_super_admin();
 	}
 
+	if ( ! $flag ) {
+		$flag = bp_group_is_admin();
+	}
+
+	if ( ! $flag ) {
+		$flag = bp_group_is_mod();
+	}
+
```

Comment:
Instead of this can we write single condition statement like 
```
if ( ! $flag ) {
     $flag = is_super_admin() || bp_group_is_admin() || bp_group_is_mod();
}
```

Reply:
on it